src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1341,6 +1341,12 @@ because xbuild doesn't support framework reference assemblies.
   <MakeDir Directories="$(MonoAndroidIntermediateAssemblyDir)" Condition=" !Exists ('$(MonoAndroidIntermediateAssemblyDir)') " />
 </Target>
 
+<Target Name="_CollectRuntimePackJarFiles">
+  <ItemGroup>
+    <AndroidJavaLibrary Include="@(ResolvedFileToPublish)" Condition=" '%(ResolvedFileToPublish.Extension)' == '.jar' " />
```

Comment:
Is the `.jar` file in all 4 runtime packs? `android-arm`, `android-arm64`, `android-x86`, android-x64`?

We'll probably have to deduplicate them somehow. Let's wait to look into this, when the changes in dotnet/runtime flow to us.

Reply:
Yes, each runtimepack should have a copy of the jar file.